docs/en/stack/security/authorization/privileges.asciidoc

```diff
@@ -62,6 +67,11 @@ who created or updated them.
 
 --
 
+`read_ccr`::
+All read only {ccr} operations, such as getting information about indices and 
```

Comment:
I would highlight that this cluster level privilege only needs to be configured in the leader cluster.

Reply:
Thanks, I've added that info in this PR.